dts/riscv/it8xxx2.dtsi

```diff
@@ -707,6 +707,8 @@
 			port-num = <1>;
 			gpios = <&gpiob 0 0>;
 			uart-dev = <&uart1>;
+			pinctrl-0 = <&uart1_rx_default &uart1_tx_default>;
+			pinctrl-names = "default";
```

Comment:
-1 on providing pin assignments at SoC level, unless it can be justified that there's only a single choice for this peripheral. Pin control defers this decision to the board, so please update IT8XXX2 based boards with the selected choice. In general, it doesn't make sense to provide such defaults because pinmux is precisely available to let board designers make routing easier, so every board will likely have its own pinmux selection.

Reply:
Done.